README.md

```diff
@@ -0,0 +1,244 @@
+
+<div style="display: flex; justify-content: center;">
+    <span style="display: inline-block; border: 5px solid #30558a; border-radius: 25px; padding: 7px;">
+        <img src="paintings/car-sharing-logo.png" alt="Car Sharing Service" style="border-radius: 10px; width: 150px">
+    </span>
+</div>
+
+
+
+## Introduction
+
+Welcome to the Car Sharing Service, an online platform for managing car rentals.
+This system simplifies the process of renting cars, managing inventory,
+and handling payments.
+
+<div style="display: flex; justify-content: center;">
+    <span style="display: inline-block; border: 5px solid #30558a; border-radius: 25px; padding: 7px;">
+        <img src="paintings/telegram-example.png" alt="Car Sharing Service" style="border-radius: 10px; width: 350px">
+    </span>
+</div>
+
+Whether you're a customer looking to rent a car or an administrator overseeing the service, this README will guide you through the key aspects of our application.
+
+
+## Table of Contents
+
+- [Technologies and Tools](#technologies-and-tools)
+- [Functionalities](#functionalities)
+- [Getting started](#getting-started)
+- [Swagger Documentation](#swagger-documentation)
+- [Postman](#postman)
+- [Controllers](#controllers)
+- [Architecture](#architecture)
+- [Project Structure](#project-structure)
+- [Contacts](#contacts)
+
+## Technologies and Tools
+
+This application is built using the following technologies and tools:
+
+- **Spring Boot**: For rapid application development.
+- **Spring Security**: Ensuring the security and authentication of users.
+- **Spring Data JPA**: Simplifying database operations.
+- **Swagger**: Providing API documentation for developers.
+- **Stripe**: Facilitating online payments.
+- **Telegram API**: Sending notifications.
+- **Docker**: Containerization for easy deployment.
+- **AWS**: Hosting the application.
+
+## Functionalities
+
+Our application offers the following functionalities:
+
+- **Car Inventory Management**: Add, update, and delete cars. Check car availability.
+- **User Profiles**: Register and manage user profiles, including role updates.
+- **Rental Management**: Rent cars by manager or user, return rentals, and view rental history.
+- **Payment Integration**: Make payments online using Stripe.
+- **Notifications**: Receive notifications about new rentals, overdue rentals, and successful payments.
+
+## Getting started
+
+To simplify deployment and management of this application, using Docker is recommended.\
+Docker allows you to build, package, and run this application in containers, making the deployment process straightforward and repeatable.
+
+### Installing Docker
+
+If Docker is not already installed on your computer, you can download it from the [official Docker website](https://www.docker.com/get-started).
+
+### Deploying the Application with Docker
+
+To deploy this application with Docker, follow these steps:
+
```

Comment:
In my opinion, we also need to mention .env, because there  are all our secrets to connect TelegramBot and StripeAPI
```
# Database Configuration
MYSQL_ROOT_PASSWORD=SuperStrongPassword
MYSQL_USER=admin
MYSQL_PASSWORD=StrongPassword1234
MYSQL_DB=car_sharing_app
MYSQL_LOCAL_PORT=3307
MYSQL_DOCKER_PORT=3306
MYSQL_PING_CHECK=mysqladmin ping -u root -p${MYSQL_ROOT_PASSWORD}

# Spring Boot Configuration
SPRING_LOCAL_PORT=8088
SPRING_DOCKER_PORT=8080
DEBUG_PORT=5005

# Telegram Bot Configuration
TELEGRAM_BOT_NAME=YourTelegramBotName
TELEGRAM_BOT_TOKEN=YourBotToken

# Stripe Configuration
STRIPE_SECRET_KEY=YourStripeSecretWord
```